common/src/fixed_hash.rs

```diff
@@ -0,0 +1,704 @@
+// Copyright 2020 Parity Technologies
+//
+// Modified in 2022 by
+//   Carla Yap <carla.yap@rbblab.com>
```

Comment:
Can you put your Mintlayer email rather than rbb please :)

Reply:
done,